src/components/components/member-training-resources/member-training-resources.js

```diff
@@ -29,7 +29,12 @@ const TrainingResource = props => {
 
   return (
     <div className={classes}>
-      <a href={url} className="member-training-resources__wrapper-link">
```

Comment:
I don't remember if this is an external link or not, could you verify that this is an external link?

Reply:
yes sure

Reply:
Its an external link, so we need that here